orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/OortService.groovy

```diff
@@ -100,20 +97,27 @@ public interface OortService {
                             @Path("type") String type,
                             @Body Map<String, ? extends Object> data)
 
-  @GET("/cache/{cloudProvider}/{type}")
+  @GET("/cache/{cloudProvider}/{entityType}")
```

Comment:
thanks for the cleanup in here, sorry for the merge conflict in my last commit...